src/Compilers/CSharp/Test/Symbol/DeclarationTests.cs

```diff
@@ -441,6 +444,11 @@ public override SyntaxTree WithFilePath(string path)
             {
                 throw new NotImplementedException();
             }
+
+            public override SyntaxTree WithDiagnosticOptions(ImmutableDictionary<string, ReportDiagnostic> options)
+            {
+                throw new NotImplementedException();
+            }
```

Comment:
Full method bodies vs. expression method bodies: both are fine. Would prefer we only use one in the same file in the same PR :)

Reply:
Sure. The reason for the difference is here I just used whatever the IDE generated, whereas the expression body code I wrote by hand.

Reply:
Haven't seen this addressed. If you don't want to address that's fine. But hard to see given the current state if you've made a decision on this. 

---
In reply to: [182517551](https://github.com/dotnet/roslyn/pull/25964#discussion_r182517551) [](ancestors = 182517551)